styles.css

```diff
@@ -3,53 +3,122 @@
   height: 100vh;
   width: 100vw;
   display: grid;
-  grid-template:
-    'header main main main main'
-    'header main main main main'
-    'header section section section section'
-    'header section section section section'
-    'header section section section section'
-    
+  grid-template-columns: 20% 80%;
+  /* grid-template-rows: 65% 35%; */
+
 }
 
 header {
-  display: grid;
-  grid-row-start: span 5;
-  border: solid;
+  margin-left: 2%;
 
 }
 
 main {
   display: grid;
   grid-row: 1 / span 2;
   grid-column: 2 / span 4;
+  /* grid-auto-rows: 35%; */
 
 }
 
 section {
   display: grid;
   grid-row: 3 / span 3;
   grid-column: 2 / span 4;
+  border: solid;
+  /* grid-auto-rows: 65%; */
 }
 
+article{
+  height: 30vh;
+  width: 25vw;
+  border: solid;
+}
+
+/* Add div class name for styling/color */
+div{
+  border: solid;
+}
 
-.idea-images,
-.search-image {
+nav{
+  grid-template-rows: 30% 70%;
+  border-top: solid;
+  border-bottom: solid;
+  font-size: 10px;
```

Comment:
to make our text responsive, we should use fr or ems rather than px for our code.